2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Configure stats pid to 162154
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Loading settings from /home/heejunyoon/.config/wandb/settings
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Loading settings from /home/heejunyoon/1_CODE/VILA/wandb/settings
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2025-04-03 17:38:18,443 WARNING MainThread:162154 [wandb_setup.py:_flush():79] Could not save program above cwd: /home/heejunyoon/anaconda3/envs/vila/bin/vila-eval
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/home/heejunyoon/anaconda3/envs/vila/bin/vila-eval', 'program': '/home/heejunyoon/anaconda3/envs/vila/bin/vila-eval'}
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_setup.py:_flush():79] Applying login settings: {}
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_init.py:_log_setup():533] Logging user logs to /home/heejunyoon/1_CODE/VILA/wandb/run-20250403_173818-vo5cujvk/logs/debug.log
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_init.py:_log_setup():534] Logging internal logs to /home/heejunyoon/1_CODE/VILA/wandb/run-20250403_173818-vo5cujvk/logs/debug-internal.log
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_init.py:init():619] calling init triggers
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'model_path': 'Efficient-Large-Model/Llama-3-VILA1.5-8B', 'conv_mode': 'auto'}
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_init.py:init():669] starting backend
2025-04-03 17:38:18,443 INFO    MainThread:162154 [wandb_init.py:init():673] sending inform_init request
2025-04-03 17:38:18,444 INFO    MainThread:162154 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-03 17:38:18,444 INFO    MainThread:162154 [wandb_init.py:init():686] backend started and connected
2025-04-03 17:38:18,445 INFO    MainThread:162154 [wandb_init.py:init():781] updated telemetry
2025-04-03 17:38:18,449 INFO    MainThread:162154 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2025-04-03 17:38:19,074 INFO    MainThread:162154 [wandb_init.py:init():867] starting run threads in backend
2025-04-03 17:38:19,137 INFO    MainThread:162154 [wandb_run.py:_console_start():2456] atexit reg
2025-04-03 17:38:19,137 INFO    MainThread:162154 [wandb_run.py:_redirect():2305] redirect: wrap_raw
2025-04-03 17:38:19,137 INFO    MainThread:162154 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-04-03 17:38:19,137 INFO    MainThread:162154 [wandb_run.py:_redirect():2395] Redirects installed.
2025-04-03 17:38:19,138 INFO    MainThread:162154 [wandb_init.py:init():911] run started, returning control to user process
2025-04-03 17:38:19,178 WARNING MsgRouterThr:162154 [router.py:message_loop():75] message_loop has been closed
